### ISSUE TITLE
Section numbers in TOC but not content

### ISSUE THREAD

**Issue by esilvia** [ASPRSorg/LAS#27]
Section numbers are getting assigned in the Table of Contents but not in the actual body text. For example, note how "Format Definition" was assigned section number 4 in the TOC, but the 4 is missing on Page 8.

**Comment by esilvia** [comment]
Related? Might be connected to the titlesec.sty warning described in #23 

If so, workaround described here: https://groups.google.com/forum/#!topic/sphinx-users/4OIOfA1tQRM

**Comment by esilvia** [comment]
Nope. That workaround did stop the warning, but it didn't fix section numbers.

**Comment by esilvia** [comment]
Turns out it's as simple as adding the `:numbered:` tag to the `toctree` directive.

http://www.sphinx-doc.org/en/stable/usage/restructuredtext/directives.html#table-of-contents

**Comment by hobu** [comment]
ooh, nice.  👍
